src/constants.ts

```diff
@@ -66,4 +67,6 @@ export const IS_AR_QUICKLOOK_CANDIDATE = (() => {
 
 // @see https://developer.chrome.com/multidevice/user-agent
 export const IS_IOS_CHROME = IS_IOS && /CriOS\//.test(navigator.userAgent);
-export const IS_IOS_SAFARI = IS_IOS && /Safari\//.test(navigator.userAgent);
\ No newline at end of file
+export const IS_IOS_SAFARI = IS_IOS && /Safari\//.test(navigator.userAgent);
+export const IS_IE =
+    !((window as any).ActiveXObject) && 'ActiveXObject' in window;
```

Comment:
👍 

This specifically detects IE11. Let's either note that in a comment or change the name to `IS_IE11`.

Reply:
🍻